great_expectations/core/util.py

```diff
@@ -411,6 +412,11 @@ def convert_to_json_serializable(  # noqa: C901, PLR0911, PLR0912
         # Connection is a module, which is non-serializable. Return module name instead.
         return "sqlalchemy.engine.base.Connection"
 
+    # Provisions for pytest and unittest.mock (to facilitate developing unit tests that utilize Mock objects).
+    if isinstance(data, mock.Mock):
+        return "mock.Mock"
```

Comment:
I don't think we should change production code for the sake of making it work better with a test - instead, we can mock the actual util in the test and scope it accordingly

Reply:
@cdkini You are right -- very good point; I disabled automerge and see how this can be solved properly.  Thanks!